tm/src/main/java/io/seata/tm/api/DefaultGlobalTransaction.java

```diff
@@ -250,6 +259,11 @@ public GlobalTransactionRole getGlobalTransactionRole() {
         return role;
     }
 
+    @Override
+    public long getBeginTime() {
```

Comment:
`createTime` is preferred on the client side and use `beginTime` on server side.